### ISSUE TITLE
Noson App crashes after launch 

### ISSUE THREAD

**Issue by Hergeirs** [janbar/noson-app#152]
I have a weird issue.
The app seems to crash a few seconds after launch no matter how I install it.

# Flatpak
When installing the app through Flatpak, this is the output I get:

```
>flatpak run io.github.janbar.noson                                                                                                               (11-19 12:32)
Signal handler is enabled.
"no file found for translations ':/i18n/noson_fo.qm' (using default)."
thumbnailer: register API [LASTFM]
thumbnailer: register API [DEEZER]
installing thumbnails cache in folder "/home/hergeir/.var/app/io.github.janbar.noson/data/janbar/io.github.janbar.noson/QML/OfflineStorage"
thumbnailer is initialized
qml: LANG=fo_FO
thumbnailer: configure API [DEEZER]
registerContent: 0x5647c00703b0 ()
registerContent: 0x5647c006d700 (FV:2)
registerContent: 0x5647c006f300 ()
registerContent: 0x5647c0069070 (SQ:)
registerContent: 0x5647c006e500 ()
registerContent: 0x5647bfe8c170 ()
scanner engine started
(SONOS)FindDeviceDescription: starting new context
(SONOS)FindDeviceDescription: location url found (http://192.168.178.37:1400/xml/device_description.xml)
(SONOS)FindDeviceDescription: server string found (Linux UPnP/1.0 Sonos/61.1-82310 (ZPS18))
(SONOS)FindDeviceDescription: search target matches
(SONOS)ParseZoneGroupState: new group 'RINCON_7828CAEA7F1A01400:2178448626' with coordinator 'RINCON_7828CAEA7F1A01400'
(SONOS)ParseZoneGroupState: new group member 'RINCON_7828CAEA7F1A01400' (Bathroom)
(SONOS)ParseZoneGroupState: topology key 153027814
systemEventCB: container [FV:2] has being updated to 1
loadContent: 0x5647c00703b0 ()
handleRenderingControlChange: sig=7 volume: 0.990 [1]
create player 1 [Bathroom]
loadContent: 0x5647bfe8c170 ()
systemEventCB: container [FV:2] has being updated to 1
handleRenderingControlChange: [RINCON_7828CAEA7F1A01400] sig=4 volume: 7.000 [7]
handleRenderingControlChange: sig=6 volume: 7.000 [7]
loadContent: 0x5647c006d700 (FV:2)
qml: Reloading the zone ...
qml: Connecting zone 'Bathroom'
Succeeded to register "org.mpris.MediaPlayer2.noson.Bathroom" on the session bus
registerContent: 0x5647c03e0fe0 ()
loadContent: 0x5647c03e0fe0 (Q:0)
```

# AUR
When installing through the AUR I get the same behaviour, but different output in terminal:

```
>noson-app
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Signal handler is enabled.
"no file found for translations ':/i18n/noson_fo.qm' (using default)."
thumbnailer: register API [LASTFM]
thumbnailer: register API [DEEZER]
installing thumbnails cache in folder "/home/hergeir/.local/share/janbar/io.github.janbar.noson/QML/OfflineStorage"
thumbnailer is initialized
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:364:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:333:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:303:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:267:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:244:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:183:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:83:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:401:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:395:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:389:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:383:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:377:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:371:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:364:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:358:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:352:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:343:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:234:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:211:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/TrackQueue.qml:41:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Player.qml:531:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qml: LANG=fo_FO
thumbnailer: configure API [DEEZER]
registerContent: 0x557117c61de0 ()
registerContent: 0x557117c5f160 (FV:2)
registerContent: 0x557117c60e50 ()
registerContent: 0x557117af4b00 (SQ:)
registerContent: 0x557117a8c000 ()
registerContent: 0x557117a8e640 ()
scanner engine started
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/Zones.qml:279:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/Zones.qml:272:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
(SONOS)FindDeviceDescription: starting new context
(SONOS)FindDeviceDescription: location url found (http://192.168.178.37:1400/xml/device_description.xml)
(SONOS)FindDeviceDescription: server string found (Linux UPnP/1.0 Sonos/61.1-82310 (ZPS18))
(SONOS)FindDeviceDescription: search target matches
(SONOS)ParseZoneGroupState: new group 'RINCON_7828CAEA7F1A01400:2178448626' with coordinator 'RINCON_7828CAEA7F1A01400'
(SONOS)ParseZoneGroupState: new group member 'RINCON_7828CAEA7F1A01400' (Bathroom)
(SONOS)ParseZoneGroupState: topology key 153027814
qrc:/controls2/components/MusicToolbar.qml:309:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:269:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
loadContent: 0x557117c61de0 ()
qrc:/controls2/components/MusicToolbar.qml:182:18: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:115:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
handleRenderingControlChange: sig=7 volume: 0.990 [1]
create player 1 [Bathroom]
systemEventCB: container [FV:2] has being updated to 1
loadContent: 0x557117a8e640 ()
qrc:/controls2/Zones.qml:196:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/SelectMusicListItem.qml:69:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/SelectMusicListItem.qml:62:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qml: Reloading the zone ...
qml: Connecting zone 'Bathroom'
Succeeded to register "org.mpris.MediaPlayer2.noson.Bathroom" on the session bus
registerContent: 0x557117c0de60 ()
loadContent: 0x557117c0de60 (Q:0)
loadContent: 0x557117c5f160 (FV:2)
playerEventCB: container [FV:2] has being updated to 1
systemEventCB: container [FV:2] has being updated to 1
qrc:/controls2/noson.qml:1261:23: QML ItemDelegate: Binding loop detected for property "height"
qrc:/controls2/components/Queue.qml:141:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingFullView.qml:268:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingFullView.qml:55:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:375:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:274:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:152:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingBubble.qml:96:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingBubble.qml:69:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingControlerView.qml:40:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qml: Set queue position view at -1
handleRenderingControlChange: [RINCON_7828CAEA7F1A01400] sig=4 volume: 7.000 [7]
handleRenderingControlChange: sig=6 volume: 7.000 [7]
[1]    51225 floating point exception (core dumped)  noson-app
```
I have tried running with `QT_QPA_PLATFORM=wayland` and it doesn't make a difference.

# PulseAudio 
Only difference I have on this laptop is that I don't use pulseaudio on this laptop.

```
>pactl info                                                                                                                                                                         (11-19 12:54)
Server String: pipewire-0
Library Protocol Version: 34
Server Protocol Version: 34
Is Local: yes
Client Index: 54
Tile Size: 4096
User Name: hergeir
Host Name: hergeirs-arch
Server Name: pulseaudio (on PipeWire 0.3.15)
Server Version: 13.99.0
Default Sample Specification: float32le 2ch 48000Hz
Default Channel Map: front-left,front-right
Default Sink: alsa_output.pci-0000:00:1f.3.analog-stereo
Default Source: alsa_input.pci-0000:00:1f.3.analog-stereo
Cookie: 6e93:a950

```

# System Info
```
>neofetch --off
OS: Arch Linux x86_64 
Host: XPS 15 7590 
Kernel: 5.9.8-arch1-1 
Uptime: 1 hour, 29 mins 
Packages: 1526 (pacman), 15 (flatpak) 
Shell: zsh 5.8 
Resolution: 3840x2160 
DE: GNOME 
WM: Mutter 
WM Theme: Flat-Remix-Blue-Darkest-fullPanel 
Theme: Flat-Remix-GTK-Blue-Darkest-NoBorder [GTK2/3] 
Icons: Flat-Remix-Blue-Dark [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel i7-9750H (12) @ 2.600GHz 
GPU: Intel UHD Graphics 630 
GPU: NVIDIA GeForce GTX 1650 Mobile / Max-Q 
```

Please tell me If there's more information I can give you.

**Comment by dantowle** [comment]
I have this same issue on 2 brand new Raspberry Pi 4 computers. One is running stock Raspian and I loaded Noson from the Raspberry Pi store. One is running Raspberry Ubuntu, and I loaded Noson every different way I could, but it closes on both computers just as the noson screen pops up. I can't find anything about an incompatibility with the arm chip. The only thing I've found on this issue is this post with no answers.  Have you solved the problem?

**Comment by janbar** [comment]
I didn't find any issue until now here. The crash is caused by divide 0 but where ?
You should run the app with "gdb" to catch the backtrace.

**Comment by Hergeirs** [comment]
Here's what I got following your instructions :)
Sorry about the huge delay.

```

$ gdb /usr/lib/noson/noson-gui                                                                                                                                                                                                                                                                                  (02-05 15:55)
GNU gdb (GDB) 10.1
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-pc-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from /usr/lib/noson/noson-gui...
(No debugging symbols found in /usr/lib/noson/noson-gui)
(gdb) run
Starting program: /usr/lib/noson/noson-gui 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
[New Thread 0x7ffff2a0c640 (LWP 1040543)]
[New Thread 0x7ffff1138640 (LWP 1040544)]
[New Thread 0x7ffff0937640 (LWP 1040545)]
[New Thread 0x7fffe3fff640 (LWP 1040546)]
[New Thread 0x7fffe37fe640 (LWP 1040547)]
Signal handler is enabled.
"no file found for translations ':/i18n/noson_fo.qm' (using default)."
[New Thread 0x7fffe2398640 (LWP 1040548)]
[New Thread 0x7fffe1b97640 (LWP 1040549)]
[New Thread 0x7fffe1396640 (LWP 1040550)]
[New Thread 0x7fffe0b95640 (LWP 1040551)]
[New Thread 0x7fffcbfff640 (LWP 1040552)]
thumbnailer: register API [LASTFM]
thumbnailer: register API [DEEZER]
installing thumbnails cache in folder "/home/hergeir/.local/share/janbar/io.github.janbar.noson/QML/OfflineStorage"
[New Thread 0x7fffca929640 (LWP 1040553)]
[New Thread 0x7fffca0f8640 (LWP 1040554)]
thumbnailer is initialized
[New Thread 0x7fffc91f0640 (LWP 1040555)]
[New Thread 0x7fffc89ef640 (LWP 1040556)]
[New Thread 0x7fffb0dd1640 (LWP 1040557)]
[New Thread 0x7fffa3fff640 (LWP 1040558)]
[New Thread 0x7fffa37fe640 (LWP 1040559)]
[New Thread 0x7fffa2ffd640 (LWP 1040560)]
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:364:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:333:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:303:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:267:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:244:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:183:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Dialog/DialogSoundSettings.qml:83:21: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:401:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:395:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:389:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:383:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:377:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:371:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:364:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:358:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:352:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:343:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:234:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/noson.qml:211:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/TrackQueue.qml:41:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Player.qml:531:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qml: LANG=fo_FO
thumbnailer: configure API [DEEZER]
registerContent: 0x555555fc0b10 ()
registerContent: 0x555555fbde00 (FV:2)
registerContent: 0x555555fbfb80 ()
registerContent: 0x555555e53370 (SQ:)
registerContent: 0x555555fbebf0 ()
registerContent: 0x555555dfff20 ()
[New Thread 0x7fffa27fc640 (LWP 1040561)]
[New Thread 0x7fffa1ffb640 (LWP 1040562)]
[New Thread 0x7fffa17fa640 (LWP 1040563)]
[New Thread 0x7fffa0ff9640 (LWP 1040564)]
[New Thread 0x7fff83fff640 (LWP 1040565)]
[New Thread 0x7fff7b7fe640 (LWP 1040566)]
[New Thread 0x7fff837fe640 (LWP 1040567)]
[New Thread 0x7fff82ffd640 (LWP 1040568)]
[New Thread 0x7fff827fc640 (LWP 1040569)]
[New Thread 0x7fff81ffb640 (LWP 1040570)]
scanner engine started
[New Thread 0x7fff817fa640 (LWP 1040571)]
[New Thread 0x7fff80ff9640 (LWP 1040572)]
[New Thread 0x7fff7bfff640 (LWP 1040573)]
[New Thread 0x7fff7affd640 (LWP 1040574)]
[New Thread 0x7fff7a7fc640 (LWP 1040575)]
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Delegates/MusicListItem.qml:39:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/Zones.qml:279:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/Zones.qml:272:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:328:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:288:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:182:18: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/MusicToolbar.qml:115:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/Queue.qml:141:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingFullView.qml:269:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingFullView.qml:55:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:375:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:274:17: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/NowPlayingToolbar.qml:152:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingBubble.qml:96:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingBubble.qml:69:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/controls2/components/RenderingControlerView.qml:40:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
parse: ERROR: bad magic header in file /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Save Me-0QCnoKEAkDk.m4a
parsing /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Save Me-0QCnoKEAkDk.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Forever Amen (Interlude)-mScAmyX6lso.m4a
parsing /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Forever Amen (Interlude)-mScAmyX6lso.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Paradigm (Interlude)-H6ZPbHRMRqw.m4a
parsing /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Paradigm (Interlude)-H6ZPbHRMRqw.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Open Over Us-4h9Yjq5aSag.m4a
parsing /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Gretzinger - Open Over Us-4h9Yjq5aSag.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Getzinger - Bright Ones-lHh1eDkNIcI.m4a
parsing /home/hergeir/Music/Steffany Gretzinger/Blackout/Steffany Getzinger - Bright Ones-lHh1eDkNIcI.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Jacob Collier/Djesse Vol. 3/Time Alone With You- Jacob Collier and Daniel Caesar (Transcription)-AoEPBdhDEnM.m4a
parsing /home/hergeir/Music/Jacob Collier/Djesse Vol. 3/Time Alone With You- Jacob Collier and Daniel Caesar (Transcription)-AoEPBdhDEnM.m4a (M4A) failed
parse: ERROR: bad magic header in file /home/hergeir/Music/Awesome/2. Raspberries - Go All The Way-ULL7apmAJTE.m4a
parsing /home/hergeir/Music/Awesome/2. Raspberries - Go All The Way-ULL7apmAJTE.m4a (M4A) failed

Thread 32 "Thread (pooled)" received signal SIGFPE, Arithmetic exception.
[Switching to Thread 0x7fff7bfff640 (LWP 1040573)]
0x00007fffc97c8fb4 in mediascanner::M4AParser::parse_mvhd(unsigned long*, _IO_FILE*, mediascanner::MediaInfo*) () from /usr/lib/noson/NosonMediaScanner/libNosonMediaScanner.so
(gdb) bt
#0  0x00007fffc97c8fb4 in mediascanner::M4AParser::parse_mvhd(unsigned long*, _IO_FILE*, mediascanner::MediaInfo*) () at /usr/lib/noson/NosonMediaScanner/libNosonMediaScanner.so
#1  0x00007fffc97c9091 in mediascanner::M4AParser::parse_moov(unsigned long*, _IO_FILE*, mediascanner::MediaInfo*) () at /usr/lib/noson/NosonMediaScanner/libNosonMediaScanner.so
#2  0x00007fffc97c94cc in mediascanner::M4AParser::parse(mediascanner::MediaFile*, mediascanner::MediaInfo*, bool) () at /usr/lib/noson/NosonMediaScanner/libNosonMediaScanner.so
#3  0x00007fffc97c4a4c in mediascanner::MediaExtractor::run() () at /usr/lib/noson/NosonMediaScanner/libNosonMediaScanner.so
#4  0x00007ffff6e33302 in  () at /usr/lib/libQt5Core.so.5
#5  0x00007ffff6e2ff0f in  () at /usr/lib/libQt5Core.so.5
#6  0x00007ffff62b93e9 in start_thread () at /usr/lib/libpthread.so.0
#7  0x00007ffff6aa0293 in clone () at /usr/lib/libc.so.6

```

**Comment by janbar** [comment]
Thanks. The issue is the M4A parser. You have probably a local file (m4a) with an invalid data crashing the app. I will fix it soon.

**Comment by janbar** [comment]
The issue is fixed with the release 4.4.8.

**Comment by Hergeirs** [comment]
Nice! thanks for the quick work :+1: 
I will test the new version out :)
